irohad/ordering/impl/single_peer_ordering_service.cpp

```diff
@@ -112,9 +112,10 @@ namespace iroha {
             // In case of restart it reloads state.
             if (persistent_state_->createOsPersistentState() |
                 [this](const auto &state) {
-                  return state->saveProposalHeight(proposal_height_);
+                  return state->saveProposalHeight(proposal_height_+1);
```

Comment:
I suggest use ```++proposal_height_ ``` here and get rid of line 118 increment.

Reply:
Not sure if that will work. If expression in that line will fail, then we will not save proposal height, and increase it. This should not happen